logs/quic.rst

```diff
@@ -0,0 +1,99 @@
+====
+QUIC
+====
+
+.. versionadded:: 6.1
+
+The QUIC protocol integrates encryption, stream multiplexing and flow control at
+the transport layer. QUIC uses TLS 1.3 by default. The QUIC spicy analyzer
+attempts to handle the QUIC INITIAL packet payload to provide greater
+observability into the protocol, including the server name.
```

Comment:
maybe shorten as "to provide visibility into the TLS handhake".

It does a few things more like retry packet handling, but yes, it's not a full protocol analyzer.

Reply:
✅